website/docs/r/logs_pipeline.html.markdown

```diff
@@ -0,0 +1,254 @@
+                }
+            }
+            category {
+                name = "verbose"
+                filter {
+                    query = "@severity: \"-\""
+                }
+            }
+        }
+    }
+    processor {
+        date_remapper {
+            name = "sample date remapper"
+            is_enabled = true
+            sources = ["_timestamp", "published_date"]
+        }
+    }
+    processor {
+        message_remapper {
+            name = "sample message remapper"
+            is_enabled = true
+            sources = ["msg"]
+        }
+    }
+    processor {
+        status_remapper {
+            name = "sample status remapper"
+            is_enabled = true
+            sources = ["info", "trace"]
+        }
+    }
+    processor {
+        trace_id_remapper {
+            name = "sample trace id remapper"
+            is_enabled = true
+            sources = ["dd.trace_id"]
+        }
+    }
+    processor {
+        service_remapper {
+            name = "sample service remapper"
+            is_enabled = true
+            sources = ["service"]
+        }
+    }
+    processor {
+        grok_parser {
+            name = "sample grok parser"
+            is_enabled = true
+            source = "message"
+            grok {
+                support_rules = ""
+                match_rules = "Rule %%{word:my_word2} %%{number:my_float2}"
+            }
+        }
+    }
+    processor {
+        pipeline {
+            name = "nested pipeline"
+            is_enabled = true
+            filter {
+                query = "source:foo"
+            }
+            processor {
+                url_parser {
+                    name = "sample url parser"
+                    sources = ["url", "extra"]
+                    target = "http_url"
+                    normalize_ending_slashes = true
+                }
+            }
+        }
+    }
+    processor {
+        user_agent_parser {
+            name = "sample user agent parser"
+            is_enabled = true
+            sources = ["user", "agent"]
+            target = "http_agent"
+            is_encoded = false
+        }
+    }
+}
+```
+## Example Usage: Datadog logs pipeline order
+
+```hcl
+resource "datadog_logs_pipelineorder" "sample_pipeline_order" {
+    name = "sample_pipeline_order"
+    depends_on = [
+        "datadog_logs_pipeline.sample_pipeline"
+    ]
+    pipelines = [
+        "${datadog_logs_pipeline.my.id}"
+    ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Your pipeline name.
+* `is_enabled` - (Optional, default = false) Boolean value to enable your pipeline.
+* `filter` - (Required) Defines your pipeline filter. Only logs that match the filter criteria are processed by this pipeline.
+  * `query` - (Required)
+* `processor` - (Optional) processors or nested pipelines. See [below](logs_pipeline.html#Processors) for more detailed descriptions.
+
+### Processors
```

Comment:
Same remark, why don't we group required and optional parameters ?

Reply:
I write this part with the same idea. In addition, `name` and `is_enabled` always comes the first (common to all processors as well), followed by the distinct attributes for different processors.